rules/scala/private/import.bzl

```diff
@@ -24,7 +24,11 @@ def scala_import_implementation(ctx):
         _jar = []
         _src_jar = []
         for jar in ctx.files.jars:
-            if jar.basename.endswith("sources.jar") or jar.basename.endswith("src.jar"):
+            if (
+                jar.basename.endswith("sources.jar") or
```

Comment:
In the other places we check for `-sources.jar` and `-src.jar` (with the `-`), should that be consistent here?

Reply:
Yep. I'll change this.